addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CodeInjectionScanRule.java

```diff
@@ -169,6 +169,13 @@ public void scan(HttpMessage msg, String paramName, String value) {
      */
     private boolean testPhpInjection(String paramName) {
         for (String phpPayload : PHP_PAYLOADS) {
+            // Check if the scan has been stopped
+            // if yes dispose resources and exit
```

Comment:
Good opportunity to remove these unnecessary comments (same for the one in the if).

Reply:
Removed the comments